packages/react-native-renderer/src/server/ReactNativeServerFormatConfig.js

```diff
@@ -209,6 +209,13 @@ export function writeStartCompletedSuspenseBoundary(
   writeChunk(destination, SUSPENSE_COMPLETE);
   return writeChunk(destination, formatID(id));
```

Comment:
@sebmarkbage  I'm a little uncertain if my change below will work since `writeStartCompletedSuspenseBoundary` adds the boundary ID for the ReactNative version but we don't create a boundary for avoidThisFallback and there's no test so I'm not sure if excluding it would break anything for the ReactNative implementation?

Reply:
This might not be needed for RN. We don't actually use it yet. It's just aspirational. You could assign an ID and pass it but just ignore it in the DOM implementation and hope it gets dead-code-eliminated.

Reply:
Alternatively, it's probably safe to drop the ID from completed and client-rendered boundaries since we don't refer to them after they're done. So we can probably drop that from all the RN functions too.